test/test_admin.py

```diff
@@ -188,30 +191,20 @@ def test_users(admin):
         'action': 'READ',
         'resource_type': 'user',
         'resource': ['username']
```

Comment:
by default when a user is created, it gets WR permission on itself. I don't know if in the past this was different, but currently it doesn't make much sense to add WRITE permission on it, as it's a default permission.